ccd-definition/CaseEventToFields/CareSupervision/manageHearings.json

```diff
@@ -371,6 +359,18 @@
     "FieldShowCondition": "hearingStartDate=\"DO_NOT_SHOW\"",
     "PageDisplayOrder": 5
   },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
moved to next page. XUI works in a weird way, if there is hidden filed and it is used on a page further in a flow then its value is gone. To keep this filed for longer, hidden field is moved a bit

Reply:
is there an e2e that needs updating because of this?